exercise/templates/exercise/_children.html

```diff
@@ -15,6 +15,7 @@
 				{{ entry.name|parse_localization }}
 			{% else %}
 				<a href="{{ entry.link }}">{{ entry.name|parse_localization }}</a>
+				{% check_exercise_extensions children entry.name %}
```

Comment:
Could this template tag be named `check_exercise_deadline_extensions`?
`check_exercise_extension` sounds like something different.

Reply:
Yes, I can check.